lib/ts/Arena.h

```diff
@@ -166,4 +167,23 @@ Arena::str_store(const char *str, size_t len)
   return mem;
 }
 
+struct Arena_NoCache : public Arena {
+  std::allocator<uint64_t> m_alloc;
+
+  inkcoreapi void *
+  alloc(size_t size, size_t alignment = sizeof(double))
+  {
+    return m_alloc.allocate((size + 1) / sizeof(uint64_t));
+  }
+  void
+  free(void *mem, size_t size)
+  {
+    m_alloc.deallocate(static_cast<uint64_t *>(mem), size);
+  }
+};
+
+#if HAVE_LIBJEMALLOC
+#define Arena Arena_NoCache
```

Comment:
Currently I'm trying to make a non-mainline Jemalloc install (which is the point of the commit) work such that we can estimate it's performance.   As it is ... the burden of #ifdef is worse than the burden of two overridden calls.